packages/next-intl/package.json

```
@@ -91,7 +91,7 @@
     "use-intl": "workspace:^"
   },
   "peerDependencies": {
-    "next": "^10.0.0 || ^11.0.0 || ^12.0.0 || ^13.0.0 || ^14.0.0",
+    "next": "^10.0.0 || ^11.0.0 || ^12.0.0 || ^13.0.0 || ^14.0.0 || ^15.0.0",
     "react": "^16.8.0 || ^17.0.0 || ^18.0.0"
```

Thank you for this PR! :smile:

Is it also possible to allow React.js v19 in peerDependencies?

```suggestion
    "react": "^16.8.0 || ^17.0.0 || ^18.0.0 || ^19.0.0 || ^19.0.0-rc"
```

---

Next.js 15 [vendors](https://nextjs.org/blog/next-15#react-19) the React 19 RC anyway, you implicitly get React 19 even if React 18 is installed 🙂.

I'll add official support for React 19 once it hits stable and is generally available, hope that makes sense!